ElrondNetwork/elrond-go
# Optimize gas used in cross dest me mbs

* Optimized gas consumption in cross shard mini blocks with destination in self shard

**Testing scenario:**

1. Start a normal system test
2. In epoch 2, OptimizeGasUsedInCrossMiniBlocks flag will be activated
3. Start txgen from this branch and with this param:
 _**txgen branch: temp-increase-gas-erc20-esdt
 txgen starting command: ./start.sh start 100 base**_
4. After 1-2 epochs we should check the logs for "gas consumed and refunded info" messages and we should have in some of
them "total gas consumed" higher than 1.5 billion, but "total gas consumed" - "total gas refunded" always lower than 1.5 billion
